src/trans-word-level/property.cpp

```diff
@@ -75,12 +79,12 @@ Function: property_obligations_rec
 static void property_obligations_rec(
   const exprt &property_expr,
   decision_proceduret &solver,
-  std::size_t current,
-  std::size_t no_timeframes,
+  mp_integer current,
+  mp_integer no_timeframes,
```

Comment:
Given that `mp_integer` is non-trivial, how about passing those as const references?

Reply:
done